assets/airports/katl.json

```diff
@@ -49,22 +49,22 @@
         }
     ],
     "fixes": {
-        "_RWY10275WLSON249": ["N33.62033", "W84.46990"],
-        "_RWY26L095HRSHL072": ["N33.64680", "W84.38172"],
-        "_RWY26L095RONII082": ["N33.64692", "W84.37733"],
-        "_RWY26L095TUANN082": ["N33.64694", "W84.39637"],
-        "_RWY26R095HRSHL075": ["N33.64996", "W84.38824"],
-        "_RWY26R095RONII083": ["N33.64955", "W84.37944"],
-        "_RWY26R095TUANN083": ["N33.64987", "W84.37988"],
-        "_RWY27L095GRITZ111": ["N33.63198", "W84.40079"],
-        "_RWY27R095GRITZ115": ["N33.63449", "W84.38239"],
-        "_RWY28095GRITZ105": ["N33.62035", "W84.39942"],
-        "_RWY8L275MPASS299": ["N33.65023", "W84.46720"],
-        "_RWY8L275SNUFY287": ["N33.64987", "W84.47031"],
-        "_RWY8R275MPASS300": ["N33.64677", "W84.46351"],
-        "_RWY8R275SNUFY288": ["N33.64666", "W84.46258"],
-        "_RWY9L275FUTBL249": ["N33.63444", "W84.47005"],
-        "_RWY9R275FUTBL251": ["N33.63172", "W84.46646"],
+        "_RWY28275WLSON249": ["N33.62033", "W84.46990"],
+        "_RWY08R095HRSHL072": ["N33.64680", "W84.38172"],
+        "_RWY08R095RONII082": ["N33.64692", "W84.37733"],
+        "_RWY08R095TUANN082": ["N33.64694", "W84.39637"],
+        "_RWY08L095HRSHL075": ["N33.64996", "W84.38824"],
+        "_RWY08L095RONII083": ["N33.64955", "W84.37944"],
+        "_RWY08L095TUANN083": ["N33.64987", "W84.37988"],
+        "_RWY09R095GRITZ111": ["N33.63198", "W84.40079"],
+        "_RWY09L095GRITZ115": ["N33.63449", "W84.38239"],
+        "_RWY10095GRITZ105": ["N33.62035", "W84.39942"],
+        "_RWY26R275MPASS299": ["N33.65023", "W84.46720"],
+        "_RWY26R275SNUFY287": ["N33.64987", "W84.47031"],
+        "_RWY26L275MPASS300": ["N33.64677", "W84.46351"],
+        "_RWY26L275SNUFY288": ["N33.64666", "W84.46258"],
+        "_RWY27R275FUTBL249": ["N33.63444", "W84.47005"],
+        "_RWY27L275FUTBL251": ["N33.63172", "W84.46646"],

```

Comment:
All of these that have `275` heading specified with the runway may be removed, since the angle of the runway is a sufficient descriptor to comply with the convention of `_RWY[runway name][fix name][fix outbound radial]`, thus yielding `_RWY27LFUTBL251`.

Reply:
done